cdap-app-fabric/pom.xml

```diff
@@ -207,8 +207,9 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.scala-tools</groupId>
-        <artifactId>maven-scala-plugin</artifactId>
+        <groupId>net.alchim31.maven</groupId>
```

Comment:
Yes. They are from the same author, just that the old one is no longer maintained.